lib/ffprober/parsers/json.rb

```diff
@@ -0,0 +1,14 @@
+module Ffprober
+  module Parsers
+    class Json
+      def initialize(json_to_parse)
+        fail ArgumentError.new('No JSON input data') if json_to_parse.nil?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.